vizro-core/src/vizro/tables/dash_aggrid.py

```diff
@@ -4,10 +4,12 @@
 
 
 @capture("action")
-def dash_ag_grid(data_frame=None):
+def dash_ag_grid(data_frame=None, **kwargs):
     """Custom AgGrid."""
     return dag.AgGrid(
         id="get-started-example-basic",
         rowData=data_frame.to_dict("records"),
-        columnDefs=[{"field": col} for col in data_frame.columns],
+        columnDefs=[{"field": col, "filter": True} for col in data_frame.columns],
+        dashGridOptions={"rowHeight": 40},
```

Comment:
Shall we set the className here as well? Similar how we do it for the data table though - so only take our default if none is given

Reply:
Yes, this is actually already part of the docs PR